nemo/collections/nlp/modules/common/tokenizer_utils.py

```diff
@@ -193,26 +190,32 @@ def get_nmt_tokenizer(
             trust_remote_code=trust_remote_code,
         )
     elif library == 'sentencepiece':
+        from nemo.collections.common.tokenizers.sentencepiece_tokenizer import SentencePieceTokenizer
```

Comment:
whats the idea behind having these lazy imports? do we want to make some packages optional?

Reply:
I profiled and looks like we spend a lot of time importing tokenizers, but eventually we only use one (at least in most cases)

Reply:
I see, the scattered imports make code look unclean do you think there is a solution to that?

maybe having a common function to do the imports or maybe doing some importlib lazyimport stuff?
I dont mind how this PR changes the import, but would definitely prefer if we can somehow clean it, not sure about how though

Reply:
I echo the concern over the scattered imports and tbh I don't see a substantially better solution offered in python. We could do importlib but would that be that much better than the current solution? It seems hard to beat the cleanness of a few imports at the top of the file, which unfortunately bear a considerable loading time cost.

The change is isolated so I'm wondering if we could permit it here. This lazy importing pattern is used frequently in NeMo-UX btw.